qcodes/dataset/sqlite_base.py

```diff
@@ -301,17 +344,11 @@ def perform_db_upgrade(conn: SomeConnection, version: int=-1) -> None:
             action(conn)
 
 
+@upgrader
 def perform_db_upgrade_0_to_1(conn: SomeConnection) -> None:
```

Comment:
shall all the upgrader infra and the upgraders move to a separate module like `sqlite_base_upgraders`?

Reply:
What should depend on what, then?

Reply:
you're right, this might introduce circular imports.